Library/Homebrew/software_spec.rb

```diff
@@ -122,9 +122,12 @@ def option(name, description = "")
         puts "Symbols are reserved for future use, please pass a string instead"
         name = name.to_s
       end
+      unless String === name
```

Comment:
Yes. I prefer `is_a?` too. But the existing lines around it are using the `===` case equals operator, so I stuck with it for local consistency.

The Ruby style guides (and the Ruby books I've been reading) say to never explicitly use `===`, and to use `is_a?` instead. If we want all the code to pass audit, we should take a pass and convert all of the `===`s.